src/modules/portfolio/components/common/tables/market-positions-table.tsx

```diff
@@ -28,21 +36,68 @@ export class MarketPositionsTable extends React.Component<MarketPositionsTablePr
 
   render() {
    
-    const { market, isMobile } = this.props;
+    const {
+      positions,
+      numCompleteSets,
+      transactionsStatus,
+      sellCompleteSets,
+      marketId,
+      isMobile,
+      extendedView
+    } = this.props;
     const { showPercent } = this.state;
-  
+
+    const pendingCompleteSetsHash = `pending-${marketId}-${numCompleteSets &&
```

Comment:
This whole sell complete sets should be in it's own component, we might need it in an other place down the road. It would also clean up the big component.